service/src/main/java/org/cbioportal/service/impl/StructuralVariantServiceImpl.java

```diff
@@ -131,4 +109,30 @@
         return countByGenes;
     }
 
+    @Override
+    public List<StructuralVariantCountByGene> getPatientCountInMultipleMolecularProfiles(
```

Comment:
It feels like we have lots of little areas in the backend where sections of code are identical except for a call or two to getPatientXXX/getSampleXXX.  Maybe its ok and introducing some design pattern to address it will only complicate the code...it just never sits well with me (I'm having a deja vu).

Reply:
Me too. we should include this in refactoring